lib/features/palette/PaletteProvider.js

```diff
@@ -75,12 +73,6 @@ PaletteProvider.prototype.getPaletteEntries = function() {
 
     function createListener(event) {
       var shape = elementFactory.createShape(assign({ type: type }, options));
-
-      if (options) {
-        var di = getDi(shape);
-        di.isExpanded = options.isExpanded;
-      }
```

Comment:
This change is not related to the PR, is it?

Reply:
Note that I explicitly set `isMarkerVisible` to `true` for the action that is created here (same file), which affects `options`.
Until now, `options` seems to be tailored to `isExpanded` and that could break when I add code for `isMarkerVisible`. I noticed that removing this code didn't have any effect at all, so I opted for removing it altogether as a simplification.

I would argue it fits the PR, but I'll add it back if you think it's unrelated or even breaking other stuff.

Reply:
OK it looks like the code you deleted is a duplication of https://github.com/bpmn-io/bpmn-js/pull/2071/files/10ca198b1d479805812eaa473d31b15a607358aa#diff-c5327c9e63a3ac25dea818bb475153d157e315968f4607b60ee6f3dba51b94c9R206

Let's keep it removed then.